W-17989215-DWextension-28-dm (#462)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-test-asserts.adoc

= dw::test::Asserts

This module contains the set of core matchers to use in your tests.

> === Example
> 
> The following example validates if a payload is of type `Object`.
> 
> ==== Source

Author: ```suggestion
===Example

This example validates if a payload is of type `Object`.

=== Source
```
